src/lib/generateDefinition.ts

```diff
@@ -334,3 +336,45 @@ async function extenderElectricityMeter(device: Zh.Device, endpoints: Zh.Endpoin
     }
     return [new Generator({extend: m.electricityMeter, args, source: `electricityMeter`})];
 }
+
+async function extenderBinaryInput(device: Zh.Device, endpoints: Zh.Endpoint[]): Promise<GeneratedExtend[]> {
+    const generated: GeneratedExtend[] = [];
+    for (const endpoint of endpoints) {
+        const source = 'binary_input';
```

Comment:
```suggestion
        const source = 'binary';
```

I think this should be `binary`, but double check if the generate definition code is OK.

Reply:
It is possible to have both clusters at the same end point. Both would end up with the same name.

Reply:
you are right :)